pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -839,8 +839,6 @@ func mainEventLoop(zedagentCtx *zedagentContext, stillRunning *time.Ticker) {
 	getconfigCtx := zedagentCtx.getconfigCtx
 	dnsCtx := zedagentCtx.dnsCtx
 
-	hwInfoTiker := time.NewTicker(3 * time.Hour)
```

Comment:
And now we switch it to 12 hours? It should be communicated explicitly, as it can be a breaking change for some users. I suggest using the old default so as not to break anything.